samples/nrf9160/modem_shell/src/gnss/gnss_shell.c

```diff
@@ -13,6 +13,8 @@
 #include "mosh_print.h"
 #include "gnss.h"
 
+#define AGPS_CMD_LINE_INJECT_MAX_LENGTH 3500
```

Comment:
This is bigger than configure CONFIG_SHELL_CMD_BUFF_SIZE in MoSh prj.conf. Does it still work with the maximum?

Reply:
Increased CONFIG_SHELL_CMD_BUFF_SIZE to 3517

Reply:
I guess this 17 comes from `gnss agps inject `. Should we have some safety margin there if one types accidentally two spaces somewhere etc? Even if that's not an issue, I'd probably align `CONFIG_SHELL_CMD_BUFF_SIZE` to 3,5kB which is 3584